### ISSUE TITLE
ERR! Invalid name: "@gulp-sourcemaps/map-sources"

### ISSUE THREAD

**Issue by craigsander** [olefredrik/FoundationPress#986]
**What did you expect to happen?**
I expected 'npm install' to configure FoundationPress

**What happened instead?**
I'm getting an error 'ERR! Invalid name: "@gulp-sourcemaps/map-sources"

Linux 3.10.0-514.16.1.el7.x86_64
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10

npm ERR! Invalid name: "@gulp-sourcemaps/map-sources"

**Comment by ChrisAynesworth** [comment]
**What did you expect to happen?**
I am having the same issue

**What happened instead?**
I too am getting the same error. I've been using FoundationPress for a while now and haven't had any install issues. Here's a bit more of my similar error:

```
npm WARN deprecated bower@1.8.0: ..psst! While Bower is maintained, we recommend yarn and webpack for new front-end projects! Please read our blog for more.
npm WARN deprecated gulp-clean-css@2.4.0: breaking changes from clean-css 4.x. Please install gulp-clean-css 3.x
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm ERR! Darwin 16.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.6

npm ERR! Invalid name: "@gulp-sourcemaps/map-sources"
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
```

**Comment by craigsander** [comment]
Okay, I figured this out, and as I suspected, it wasn't FoundationPress, it was was one of the dependencies, specifically, gulp-sourcemaps. 

In package.json, I placed a '^' before the version of gulp-sourcemaps. That did it for me. Keep in mind, I started from scratch prior to doing this. 

"gulp-sourcemaps": "1.6.0", --> "gulp-sourcemaps": "^1.6.0",

**Comment by ChrisAynesworth** [comment]
That's weird because my version of FoundationPress has that correction, and yet I'm still getting the same error as before...

**Comment by evansoundview** [comment]
Same issue. Any luck Chris?

**Comment by olefredrik** [comment]
gulp-sourcemaps are now updated to latest version https://github.com/olefredrik/FoundationPress/pull/1018. Please check if you're able to recreate your issue after updating. You'll need to do a `npm install` after updating gulp-sourcemaps in package.json.

**Comment by ChrisAynesworth** [comment]
Hey sorry to open this up again but I found something interesting. I have still had this error with gulp-sourcemaps upon initial npm install. I didn't want to comment on this without new information though. I have tried the latest build up to today, which contains this in package.json:

`"gulp-sourcemaps": "1.6.0", --> "gulp-sourcemaps": "^1.6.0",
`
That didn't work! But oddly changing it to this works:

`"gulp-sourcemaps": "1.6.0", --> "gulp-sourcemaps": "2.4.1",`

I found this out from this link: 
[https://github.com/gulp-sourcemaps/gulp-sourcemaps/issues/298#issuecomment-291447038](url)

I'm not quite sure what the other commenter is asserting here, so am wondering why this is working!
